packages/react-components/react-nav-preview/stories/NavItem/NavItemDefault.stories.tsx

```diff
@@ -1,4 +1,49 @@
 import * as React from 'react';
-import { NavItem, NavItemProps } from '@fluentui/react-nav-preview';
+import { Nav, NavItem, NavItemProps } from '@fluentui/react-nav-preview';
+import { Folder20Regular, Folder20Filled, bundleIcon } from '@fluentui/react-icons';
+import { makeStyles, tokens } from '@fluentui/react-components';
 
-export const Default = (props: Partial<NavItemProps>) => <NavItem value={'someValue'} {...props} />;
+const Folder = bundleIcon(Folder20Filled, Folder20Regular);
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '260px',
+    rowGap: '60px',
+    paddingTop: '48px',
```

Comment:
Use `...shorthands.padding()` for all the padding values?

Reply:
Updated